packages/server/lib/modes/record.js

```diff
@@ -784,6 +784,8 @@ const createRunAndRecordSpecs = (options = {}) => {
         })
 
         if (response === responseDidFail) {
+          debug('Response did fail equals response, allowing browser to hang until it is killed: Response %o', { responseDidFail })
```

Comment:
Found this hard to read as a sentence, suggest keeping the variable name all together (I don't know how backticks work in `debug`, but you get the idea):

```suggestion
          debug('`responseDidFail` equals `response`, allowing browser to hang until it is killed: Response %o', { responseDidFail })
```